lib/readline.js

```diff
@@ -258,7 +258,7 @@ Object.defineProperty(Interface.prototype, 'columns', {
   configurable: true,
   enumerable: true,
   get: function() {
-    var columns = Infinity;
+    let columns = Infinity;
     if (this.output && this.output.columns)
       columns = this.output.columns;
     return columns;
```

Comment:
nit: how about avoiding the variable `columns` altogether?
```javascript
get: function() {
    if (this.output && this.output.columns)
      return this.output.columns;
    return Infinity;
}
```

Reply:
Hi @trivikr 

Thanks for reviewing this too.  
I did not allow myself to do these changes as the scope of the PR was limited to the `var` replacement, but I'm glad that you suggested it!  
I amended my commit and did the change :smile: